arch/arc/core/reset.S

```diff
@@ -83,7 +83,7 @@ SECTION_FUNC(TEXT,__start)
 	sr r0, [_ARC_V2_IRQ_VECT_BASE]
 #endif
 
-#if defined(CONFIG_USERSPACE)
+#ifdef CONFIG_USERSPACE
```

Comment:
Hi Yuguo,
I'm wondering how is this change related to this commit/PR topic?

If it is cleanup, could you please move it to separate patch, thanks.